l10n-pocket/en/nav.ftl

```diff
@@ -31,3 +32,19 @@ pocket-mobile-nav-aria-label = { -brand-name-pocket } Mobile Menu
 pocket-mobile-nav-close-tooltip = Close
 pocket-mobile-nav-close-menu = Close the { -brand-name-pocket } Mobile Menu
 pocket-mobile-nav-beta = Beta
+
+### Platform navigation is used in some pages:
+
+### Pocket homepage https://www-dev-pocket.allizom.org
+### Add page URL https://www-dev-pocket.allizom.org/Add
+### All platform pages https://www-dev-pocket.allizom.org/ios
+### https://www-dev-pocket.allizom.org/android
+### https://www-dev-pocket.allizom.org/chrome
+### https://www-dev-pocket.allizom.org/safari
+### https://www-dev-pocket.allizom.org/opera
+### https://www-dev-pocket.allizom.org/edge
+
+# pocket/includes/platform-nav.html
+pocket-nav-how-to-save = How to Save
```

Comment:
@alexgibson Referencing your [comment in the other PR](https://github.com/mozilla/bedrock/pull/11579#pullrequestreview-964338428), I thought it would actually be best if I include the `platform-nav.html` and `platform-footer.html` strings into `nav.ftl` and `footer.ftl` respectively, because most of the strings have already been defined in these files, and it would be repetitive if I were to use those strings in another file.
Even if I created new files and only defined the strings that haven't been defined yet, they would have _very_ little content. So I instead decided on commenting which URLs are using the `platform-nav` and `platform-footer` files to create a bit of separation in the Fluent files.

Happy to discuss if you think otherwise!

Reply:
Makes sense to me 👍